packages/marshaller/src/ddl2/dashboard.css

```diff
@@ -2,6 +2,13 @@
     border-color:red;
 }
 
-.p-TabBar-tab.p-mod-current.active {
+.p-TabBar-tab.p-mod-current.active,
+.p-TabBar.active .p-TabBar-content {
     border-color:red;
-}
\ No newline at end of file
+}
+.p-TabBar .p-TabBar-content{
+    cursor: pointer;
```

Comment:
Not sure it should be a pointer for the entire tab bar (rather than an individual tab?)

Reply:
It's only "pointer" while hovering over a tab (not the space to the right of the tabs).